core/common/src/main/java/com/dhc/common/ColorUtil.kt

```diff
@@ -0,0 +1,15 @@
+package com.dhc.common
+
+import androidx.compose.ui.graphics.Color
+import androidx.core.graphics.toColorInt
+
+/**
+ * string Hex to Color
+ */
+fun hexToColor(hex: String): Color {
+    return try {
+        Color(hex.toColorInt())
+    } catch (e: IllegalArgumentException) {
+        Color(0xFFD7E1EE)
```

Comment:
```suggestion
fun hexToColor(hex: String, defaultColor: Color): Color {
    return try {
        Color(hex.toColorInt())
    } catch (e: IllegalArgumentException) {
        defaultColor
```
Nit : 호오옥시 Color 에 들어가는 값이 하드코딩이라서 불편하다면,, 이런식으로 defaultColor 를 파라미터로 받는것도 방법일지두?!

Reply:
넘 좋아!! 안그래도 이부분 고민했어...하드코딩보다는 정의해둔거 쓰는게 좋을 것 같앗는데 변수도 추가하고 변수에 기본값도 넣어둘게!!

Reply:
디폴트 변수에 기본값 넣는것까지 너무 좋아요~~